src/cloud-api-adaptor/Dockerfile

```diff
@@ -37,8 +38,25 @@ COPY cloud-api-adaptor/proto ./proto
 
 RUN CC=gcc make ARCH=$TARGETARCH COMMIT=$COMMIT VERSION=$VERSION RELEASE_BUILD=$RELEASE_BUILD cloud-api-adaptor
 
+FROM builder-release AS iptables
+
+ARG TARGETARCH
+
+WORKDIR /iptables
+RUN --mount=type=bind,target=/versions.yaml,source=cloud-api-adaptor/versions.yaml,readonly \
```

Comment:
This format is new to me. Do you have any reference on using `--mount` with RUN? 
Is it docker specific or can work with podman also ?

Reply:
We can specify cache mounts and bind mounts to a RUN instruction in a Dockerfile.

https://docs.docker.com/build/guide/mounts/

Podman supports the feature as well.

https://github.com/containers/common/blob/10ced654d57245a78d1b2c3746c5fd99209370bd/docs/Containerfile.5.md?plain=1#L102